app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Band.scala

```diff
@@ -0,0 +1,66 @@
+package com.azavea.rf.datamodel
+
+import akka.http.scaladsl.unmarshalling._
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+import java.util.UUID
+
+case class Band(
+  id: UUID,
+  image: UUID,
+  name: String,
+  number: Int,
+  wavelength: List[Int]
+)
+
+object Band {//extends RangeUnmarshaler{
```

Comment:
Since we've gone with `List[Int]` instead of `Range` all the way, let's get rid of this comment too.